src/test/java/api/support/fixtures/LoansFixture.java

```diff
@@ -104,6 +107,15 @@ public Response attemptToCreateLoan(
     return response;
   }
 
+  public IndividualResource declareItemLost(
+    DeclareItemLostRequestBuilder builder) {
```

Comment:
Does this take it over the 80 character limit?

Reply:
It is 85.

Reply:
Ok, I guess we’ll leave it, at least till we move to the Google code style